packages/realm/src/platform/node/index.ts

```diff
@@ -18,5 +18,6 @@
 
 import "./fs";
 import "./device-info";
+import "./sync-proxy-config";
```

Comment:
Should this be done in `react-native` as well?

Reply:
Alternatively, we could just remove the `react-native` implementation and let it throw.